### PR TITLE
docs: add clarity on how to use bullseye layout 

### PR DESCRIPTION
**Pull request by starryeyez024**
## Component name

- pfe-styles

### Preview

- https://deploy-preview-1136--happy-galileo-ea79c4.netlify.app/elements/pfe-styles/demo/

### What has changed and why
 
- the bullseye container has a height of 100% which means it should be the only child in the parent container. Updated docs / demo to show this. 

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [x] Repository compiles and tests pass.
- [x] Documentation (README.md, WHY.md, etc.) updated or added.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Comment by castastrophe** [comment]
To center an item inside a container, I think it makes sense to make it 100% or flex-grow: 1 if that container is set to flex too.

File: CHANGELOG-prerelease.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 - [4cda080](https://github.com/patternfly/patternfly-elements/commit/4cda080dfc6f3b8e500712ddbe01fada3ce16e3e) feat: Launch specific component demo page when specified on `npm run start`; add start to dev command
 - [0354352](https://github.com/patternfly/patternfly-elements/commit/03543523064f4a885b44bd334b0e24a77f00aee9) feat: Open a pull request from the command-line
 - [](https://github.com/patternfly/patternfly-elements/commit/) Migrate from Travis to GitHub Actions
+- []() fix: improve demo & docs on bullseye layout
```
Comment (castastrophe) [inline CHANGELOG-prerelease.md:6]: This is now a doc update so we don't need a changelog update

Comment (starryeyez024) [reply CHANGELOG-prerelease.md]: removed

**Review by markcaron — APPROVED** [review]
Los Guapos Tomamos Mezcal

**Comment by castastrophe** [comment]
Be sure to run it through the browsers before merging. I don't see the checklist in the PR.

**Comment by markcaron** [comment]
I did. The bullseye seemed to work fine on Edge, 11, FF, Chrome and Safari. I don't see a checklist either, am I missing any browsers you all check specifically beyond "the big 5"?

**Comment by starryeyez024** [comment]
Since no code changed in this PR (docs only) it was exempt from browser testing, so I removed the normal list of browsers.

**Comment by castastrophe** [comment]
Yeah I saw that, that's why I deleted the comment.  I was out-of-date with the direction the PR took.

**Pull request merged**